OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/SsaiPlayerActivity.java

```diff
@@ -0,0 +1,84 @@
+package com.ooyala.sample.players;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import com.ooyala.android.OoyalaPlayer;
+import com.ooyala.android.PlayerDomain;
+import com.ooyala.android.analytics.IqConfiguration;
+import com.ooyala.android.configuration.Options;
+import com.ooyala.android.skin.OoyalaSkinLayout;
+import com.ooyala.android.skin.OoyalaSkinLayoutController;
+import com.ooyala.android.skin.configuration.SkinOptions;
+import com.ooyala.android.ssai.OoyalaSsaiManager;
+import com.ooyala.sample.R;
+
+import org.json.JSONObject;
+
+
+public class SsaiPlayerActivity extends AbstractHookActivity {
+    private OoyalaSsaiManager ssaiManager;
+    private String playerParams = "";
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Bundle extras = getIntent().getExtras();
+        if (extras != null) {
+            playerParams = extras.getString("player_params");
+        }
+        setContentView(R.layout.player_skin_simple_layout);
+        completePlayerSetup(asked);
+    }
+
+    @Override
+    void completePlayerSetup(boolean asked) {
+        if (asked) {
+            // Get the SkinLayout from our layout xml
+            skinLayout = (OoyalaSkinLayout) findViewById(R.id.ooyalaSkin);
+            // Create the OoyalaPlayer, with some built-in UI disabled
+            PlayerDomain playerDomain = new PlayerDomain(domain);
+            // Create an IQConfiguration class
+            IqConfiguration iqConfiguration = new IqConfiguration.Builder().setPlayerID("SSAI Android Player ID").build();
+
+            Options options = new Options.Builder().setIqConfiguration(iqConfiguration).setShowNativeLearnMoreButton(false).setShowPromoImage(false).setUseExoPlayer(true).build();
+            player = new OoyalaPlayer(pcode, playerDomain, options);
+            //Create the SkinOptions, and setup React
+            JSONObject overrides = createSkinOverrides();
+            SkinOptions skinOptions = new SkinOptions.Builder().setSkinOverrides(overrides).build();
+            playerLayoutController = new OoyalaSkinLayoutController(getApplication(), skinLayout, player, skinOptions);
+            //Add observer to listen to fullscreen open and close events
+            playerLayoutController.addObserver(this);
+            player.addObserver(this);
+
+            ssaiManager = new OoyalaSsaiManager.Builder(player).usePlayerParams(playerParams).build();
+
+            if (player.setEmbedCode(embedCode)) {
+            } else {
+                Log.e(TAG, "Asset Failure");
+            }
+        }
+    }
+
+    /**
+     * Create skin overrides to show up in the skin.
+     * Default commented. Uncomment to show changes to the start screen.
```

Comment:
This line is wrong because it's not commented out. Also please change it to read:
Commented out by default. Uncomment to show changes to the start screen.

Reply:
The comment is bad because it doesn't just applies to the start screen but it modifies the Skin.json completely. I think that for SSAI we do want to modify it. I see that he shows the control bar in the adScreen.

@orobles-ooyala please confirm

Reply:
Yes, the comment was bad, I forgot to remove it when updating the adScreen. Should be fixed now.